app/controllers/users_controller.rb

```diff
@@ -269,7 +269,7 @@ def username
     params.require(:new_username)
 
     if clashing_with_existing_route?(params[:new_username]) ||
-         User.reserved_username?(params[:new_username])
+         (User.reserved_username?(params[:new_username]) && !current_user.admin?)
```

Comment:
Just curious if we can use the `UsernameCheckerService` or at least centralise the logic in a single class.

Reply:
Doing this causes headaches with plugins, just leaving as is for now

Reply:
Ah damn it wasn't this change, it was the rest...looking